### PR TITLE
adding video adds cp

### PR DESCRIPTION
**Pull request by esnyder232**

### CODE REVIEW

**Review by KamalJDavis — APPROVED** [review]
Looks good!

File: src/channel/coolholepoints.js [hunk 1]
```diff
@@ -432,6 +432,8 @@ class Coolpoints extends ChannelModule {
    */
   spend(user, action) {
     try {
+      if(!user.channel.is(Flags.C_REGISTERED) || !user.is(Flags.U_REGISTERED) || !user.is(Flags.U_LOGGED_IN)) return;
```
Comment (cck37) [inline src/channel/coolholepoints.js:435]: nit: probably worth writing a small helper function to avoid repeating code

**Review by cck37 — APPROVED** [review]
Noice

**Review by streeeggs — APPROVED** [review]
Wrong account. Oh well

**Pull request merged**